.ruby-version

```diff
@@ -1 +1 @@
-3.2
+3.2.4
```

Comment:
This gets me everytime! I use `rbenv` on my mac, and have to update this but don't check it in. I should set an alias on my machine for `3.2` but pinning to whatever is used in the Dockerfile is more sensible 👍🏾